stream-dbs-clients/pom.xml

```diff
@@ -60,6 +60,11 @@
             <artifactId>springfox-core</artifactId>
             <version>3.0.0</version>
         </dependency>
+        <dependency>
```

Comment:
we don't need this dependency declared here, you can remove it and include the identity-bom in the dependencyManagement section similar to what we do with dbs.